CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.6
+
+- Save two months of cvs files instead of only current month.
+- Added new env var SEND_ONLY_FULL_PREVIOUS_MONTH to manage if we send data from previous month only if we have all the data for the previous month.
```

Comment:
I'm not sure we need this variable, we don't want to overcomplicate the configurations. Maybe it makes sense to just make this the default behavior, but add a small comment in the INCLUDE_PREVIOUS_MONTH description?